serialization.md

```diff
@@ -327,7 +327,6 @@ Note: whitespace has been added to this example for clarity. Whitespace is OPTIO
         <ul>
           <li>
             <code>type</code> is usually set to <code>layers</code>.
-	    There is also a Windows-specific value <code>layers+base</code> that allows a base layer to be specified in a field of <code>rootfs</code> called <code>base_layer</code>.
```

Comment:
> If the intention here is to only allow the layers-type rootfses, I
don't see a problem with dropping the field from the spec.

I'm not sure that you have full knowledge to evaluate the usage of this field, nor is this PR the forum for that discussion.

These are two _completely_ different components operating at separate layers of the specification. I don't know what you want me to say.